sympy/diffgeom/diffgeom.py

```diff
@@ -1,6 +1,8 @@
 from __future__ import print_function, division
 
 from itertools import permutations
+import itertools
```

Comment:
this should probably not be used, as you can already find stuff implemented in `sympy.tensor.array`